### PR TITLE
WIP: Re-enable Lottie

### PR DESCRIPTION
**Pull request by flozia**
🚧

### CODE REVIEW

**Review by bakulf — APPROVED** [review]

File: .github/workflows/macos-build.yaml [hunk 1]
```diff
@@ -263,6 +263,7 @@ jobs:
             CONFIG-=static \
             CONFIG+=balrog \
             MVPN_MACOS=1 \
+            lottie/lottie.pro \
```
Comment (bakulf) [inline .github/workflows/macos-build.yaml:266]: this is not needed.

File: lottie/lottie.pri [hunk 2]
```diff
@@ -15,9 +15,9 @@ HEADERS += $$PWD/lib/lottie.h \
...
            $$PWD/lib/lottiestatus.h
 
-RESOURCES += $$PWD/lib/lottie.qrc
-
 CONFIG += qmltypes
 
 QML_IMPORT_NAME = vpn.mozilla.lottie
 QML_IMPORT_MAJOR_VERSION = 1
+
+RESOURCES += $$PWD/lib/lottie.qrc
```
Comment (bakulf) [inline lottie/lottie.pri:23]: why this change?

File: lottie/lottie.pro [hunk 3]
```diff
@@ -5,4 +5,4 @@
 TEMPLATE = subdirs
 SUBDIRS += bin
 SUBDIRS += tests/unit
-SUBDIRS += tests/qml
+SUBDIRS += tests/qml
```
Comment (bakulf) [inline lottie/lottie.pro:8]: why this change?

File: mozillavpn.pro [hunk 4]
```diff
@@ -14,6 +14,7 @@ TEMPLATE = subdirs
 
 SUBDIRS += src
 SUBDIRS += tests/unit
+SUBDIRS += lottie
```
Comment (bakulf) [inline mozillavpn.pro:17]: this should not be needed.

**Review by bakulf — CHANGES_REQUESTED** [review]

**Comment by flozia** [comment]
Closing in favour of PR #2684
